src/main/java/soot/toolkits/graph/ExceptionalUnitGraph.java

```diff
@@ -322,8 +325,8 @@ protected void initialize(ThrowAnalysis throwAnalysis, boolean omitExceptingUnit
    * @return a <code>Map</code> which whose contents are equivalent to the input <code>map</code>, plus the information that
    *         <code>u</code> throws <code>caught</code> to <code>t</code>.
    */
-  private Map<Unit, Collection<ExceptionDest>> addDestToMap(Map<Unit, Collection<ExceptionDest>> map, Unit u, Trap t,
-      ThrowableSet caught) {
+  protected Map<Unit, Collection<ExceptionDest>> addDestToMap(Map<Unit, Collection<ExceptionDest>> map, Unit u,
```

Comment:
Make it protected to allow a subclass to use and/or override it. There is no reason for it to remain private.